runtime/opt/taupage/init.d/10-prepare-disks.py

```diff
@@ -29,14 +30,52 @@ def zone():
     return boto.utils.get_instance_metadata()['placement']['availability-zone']
 
 
+def retry(func):
+    def wrapped(*args, **kwargs):
+        count = 0
+        while True:
+            try:
+                return func(*args, **kwargs)
+            except boto.exceptions.BotoServerError as e:
+                if str(e.error_code) not in ('Throttling', 'RequestLimitExceeded'):
+                    raise
+                logging.info('Throttling AWS API requests...')
+
+            if count >= 10:
+                break
+            time.sleep(2 ** count * 0.5)
```

Comment:
So this sleeps up to roughly 1 second in total?  Not sure that's reliable enough.